src/types/express/request.d.ts

```diff
@@ -0,0 +1,14 @@
+import { GrowthBook } from "@growthbook/growthbook"
+
+import { FeatureFlags } from "@root/types/featureFlags"
+
+// to make the file a module and avoid the TypeScript error
```

Comment:
nit: added more details in comment for the nature of the error we are trying to avoid here 

```suggestion
// Empty export to make this file a module therefore allow global augmentation (avoiding the related error)
```

Reply:
updatedd